beacon-chain/utils/shuffle.go

```diff
@@ -3,37 +3,60 @@ package utils
 
 import (
 	"errors"
+	"math"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/prysmaticlabs/prysm/shared/hashutil"
 	"github.com/prysmaticlabs/prysm/shared/params"
 )
 
 // ShuffleIndices returns a list of pseudorandomly sampled
-// indices. This is used to use to select attesters and proposers.
-func ShuffleIndices(seed common.Hash, validatorList []uint32) ([]uint32, error) {
-	// Since we are consuming 3 bytes of entropy at a time in the loop,
+// indices. This is used to shuffle validators on ETH2.0 beacon chain.
+func ShuffleIndices(seed common.Hash, indicesList []uint32) ([]uint32, error) {
+	// Each entropy is consumed from the seed in randBytes chunks.
+	randBytes := int(params.BeaconConfig().RandBytes)
+	upperBound := uint64(math.Pow(2, float64(randBytes) * 8)) - 1
+
+	// Since we are consuming randBytes of entropy at a time in the loop,
 	// we have a bias at 2**24, this check defines our max list size and is used to remove the bias.
```

Comment:
2^(randBytes*8)

Reply:
btw 2 ^ (24) is 26 

I ended up using `1<<(randBytes*8)`